examples/cms-wordpress/jsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "@/components/*": ["components/*"],
+      "@/lib/*": ["lib/*"],
+      "@/styles/*": ["styles/*"]
+    }
+  }
+}
```

Comment:
I don't think the example needs this, this is more user preference than something that needs to be there.

Reply:
I used this in the [DatoCMS example](https://github.com/zeit/next.js/pull/13008), so thought I'd bring it over. If it's not essential, I'll remove it.